src/Compilers/VisualBasic/BasicAnalyzerDriver/VisualBasicDeclarationComputer.vb

```diff
@@ -119,35 +139,48 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         For Each decl In typeBlock.Members
                             ComputeDeclarationsCore(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken)
                         Next
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken))
+                        Dim attributes = GetAttributes(typeBlock.BlockStatement.AttributeLists)
+                        builder.Add(GetDeclarationInfo(model, node, getSymbol, attributes, cancellationToken))
                         Return
                     End If
 
                     Dim typeStatement = TryCast(node, TypeStatementSyntax)
                     If typeStatement IsNot Nothing Then
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken))
+                        Dim attributes = GetAttributes(typeStatement.AttributeLists)
+                        builder.Add(GetDeclarationInfo(model, node, getSymbol, attributes, cancellationToken))
                         Return
                     End If
 
                     Dim methodBlock = TryCast(node, MethodBlockBaseSyntax)
                     If methodBlock IsNot Nothing Then
                         Dim paramInitializers = GetParameterInitializers(methodBlock.BlockStatement.ParameterList)
-                        Dim codeBlocks = paramInitializers.Concat(methodBlock)
+                        Dim attributes = GetAttributes(methodBlock.BlockStatement.AttributeLists)
```

Comment:
>Dim attributes = GetAttributes(methodBlock.BlockStatement.AttributeLists) [](start = 24, length = 73)

Aren't we doing this below?

Reply:
[https://github.com/dotnet/roslyn/issues/23347](https://github.com/dotnet/roslyn/issues/23347)

---
In reply to: [152419184](https://github.com/dotnet/roslyn/pull/23318#discussion_r152419184) [](ancestors = 152419184)